store/postgres/migrations/000020_add_job_dependencies_table.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE job DROP IF EXISTS external_dependencies;
```

Comment:
We should be dropping table `job_dependency`.

Reply:
updated